Mathlib/SetTheory/Ordinal/Basic.lean

```diff
@@ -413,6 +413,9 @@ theorem eq_zero_or_pos : ∀ a : Ordinal, a = 0 ∨ 0 < a :=
   eq_bot_or_bot_lt
 #align ordinal.eq_zero_or_pos Ordinal.eq_zero_or_pos
 
+theorem pos_of_gt {o p : Ordinal} (h : p < o) : 0 < o :=
```

Comment:
Ah... I did this but it's causing build errors in some files that use `pos_of_gt`, which don't show up for me locally. Sorry, I'm not sure how to deal with this, what should I do here?

Reply:
Don't remove the existing theorem, instead make a new one `pos_of_gt'`. Since this is a change whose relevancy spans the entire library, it's better to put it in a separate PR.